app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -39,12 +39,16 @@ def get_routes(inventory)
         process_collection(inventory["route"], path_for_entity("route")) { |n| parse_route(n) }
       end
 
+      # Merge into results of parse_namespace
       def get_projects(inventory)
-        key = path_for_entity("project")
-        inventory["project"].each { |item| parse_project(item) }
-
-        @data[key].each do |ns|
-          @data_index.store_path(key, :by_name, ns[:name], ns)
+        key = path_for_entity("namespace")
```

Comment:
it's the same, both return `:container_projects`.
I wanted to emphasize that we're merging into same place that kubernetes parser `get_namespaces` fills...

I'm wondering if renaming this method to say `merge_projects_into_namespaces` would be good.